R/init.R

```diff
@@ -57,9 +57,24 @@ copy_assets <- function(pkg = ".") {
   pkg <- as_pkgdown(pkg)
   template <- purrr::pluck(pkg$meta, "template", .default = list())
 
+  # Copy logo if it exists
+
+  if (pkg$has_logo) {
+    file.copy(
+      find_logo(pkg$src_path),
+      file.path(pkg$dst_path, "package-logo.png")
+    )
+  }
+
   # Copy default assets
   if (!identical(template$default_assets, FALSE)) {
-    copy_asset_dir(pkg, path_pkgdown("assets"))
+    copy_asset_dir(
```

Comment:
Can this all fit on one line? 

I think in general you want to avoid excessive vertical spacing since there's only so many lines that you can show on screen at a time.